app/views/components/gds/button.scala.html

```diff
@@ -0,0 +1,35 @@
+@*
```

Comment:
Do we really need this partial? It seems to me that it does not help compared to the direct use of `@govukButton`.